docs/general/releases.md

```diff
@@ -4,17 +4,6 @@ Revised: 2022-09-08
 
 This document describes the available zot images for the various supported hardware and software platforms, along with information about image variations, image locations, and image naming formats.
 
```

Comment:
Make it clear that you can download native binaries and also docker images
For MacOS/docker image, you have to use "linux-arm64" for arm silicon and "linux-amd64" for intel silicon